reports/tasks.py

```diff
@@ -15,11 +15,16 @@
 # You should have received a copy of the GNU General Public License
 # along with Patchman. If not, see <http://www.gnu.org/licenses/>
 
+from __future__ import unicode_literals, absolute_import
+
 from django.conf import settings
 
+from reports.models import Report
+
 if settings.USE_ASYNC_PROCESSING:
     from celery.task import task
```

Comment:
First of all, Nice work on this!

This is not working like it should. should be changed in something like:
```

    from patchman.celery import app as celery_app

    @celery_app.task
```

Reply:
Oh, ok. Which version of django and celery are you using?

The above version worked for me with django 1.11 and celery 4.3.0.

Did your change work for you?

Reply:
@stokkie90 I found this is due to selinux, on CentOS 7.

You can either set `setenforce 0` to turn off selinux, or allow httpd to connect to the rabbitmq server port, 5672:

```
semanage port -a -t http_port_t -p tcp 5672
```

Both your code and mine work as expected in this case.